gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/HadoopFileSystemTestBase.java

```diff
@@ -275,6 +275,72 @@ protected void validateListFileInfo(
     }
   }
 
+  @Test
+  public void testInputStreamReadStorageStatistics() throws IOException {
+
+    // Write an object.
+    URI path = GoogleCloudStorageFileSystemIntegrationTest.getTempFilePath();
+    Path hadoopPath = ghfsHelper.castAsHadoopPath(path);
+    String text = "Hello World!";
+    int numBytesWritten = ghfsHelper.writeFile(hadoopPath, text, 1, /* overwrite= */ false);
+    GhfsStorageStatistics StorageStats = ((GoogleHadoopFileSystem) ghfs).getStorageStatistics();
+    try (FSDataInputStream readStream = ghfs.open(hadoopPath)) {
+
+      //  Check the statistics of method read().
+      readStream.read();
+      assertThat(StorageStats.isTracked("stream_read_operations")).isTrue();
+      assertThat(StorageStats.isTracked("stream_read_bytes")).isTrue();
+
+      // Check the statistics of method read(buf, off, len)
+      byte[] readbuffer = new byte[1];
+      readStream.read(readbuffer, 0, 1);
+      assertThat(StorageStats.isTracked("stream_read_operations")).isTrue();
+      assertThat(StorageStats.isTracked("stream_read_bytes")).isTrue();
+
+      // Check the statistics of method read(pos, buf, off, len)
+      byte[] readbuffer1 = new byte[2];
+      readStream.read(2, readbuffer1, 0, 2);
```

Comment:
please assert values

Reply:
Updated

Reply:
Shouldn't the value increase with new calls ?